src/components/entry/EntryOverview.js

```diff
@@ -120,7 +120,8 @@ const EntryOverview = ({ locationData, className }) => {
     </TagList>
   )
 
-  const allTypeNames = locationData && locationData.type_names.join(', ')
+  const allTypeNames =
+    locationData && locationData.type_ids.map(getTypeName).join(', ')
```

Comment:
could be more concise with optional chaining

Reply:
Clever girl!